com.synology.SynologyDrive.metainfo.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.synology.SynologyDrive</id>
+  <metadata_license>MIT</metadata_license>
+  <project_license>LicenseRef-proprietary=https://code.visualstudio.com/license</project_license>
```

Comment:
Is the license url really the correct one? :-)

Reply:
sorry... it's not the correct license url. i used visual studio code as a template and i missed that line. sorry.
it's now corrected in the new commits

Reply:
No problem, that's what the new app review process is for. :-) Anyway, the license url is still not correct.

Reply:
i've the good url in the new commit : 43e61b0a7e50cabddac1cce3d2dd037e52bf61da

Reply:
i think it's OK no, in the commit effbcde
(the good url is [https://www.synology.com/en-global/company/legal/terms_EULA](https://www.synology.com/en-global/company/legal/terms_EULA))

Reply:
Yep, it's fine now.